python/lsst/sconsUtils/scripts.py

```diff
@@ -102,15 +109,26 @@ def initialize(cls, packageName, versionString=None, eupsProduct=None, eupsProdu
             except TypeError:
                 pass
             state.targets["version"] = state.env.VersionModule(versionModuleName)
+        scripts = []
         for root, dirs, files in os.walk("."):
             if "SConstruct" in files and root != ".":
                 dirs[:] = []
                 continue
             dirs[:] = [d for d in dirs if not d.startswith('.')]
             dirs.sort()  # happy coincidence that include < libs < python < tests
             if "SConscript" in files:
-                state.log.info("Using Sconscript at %s/SConscript" % root)
-                SConscript(os.path.join(root, "SConscript"))
+                scripts.append(os.path.join(root, "SConscript"))
+        if sconscriptOrder is None:
+            sconscriptOrder = ("lib", "python", "tests", "examples", "doc")
```

Comment:
What happens when someone specifies ["python", "foo", "tests"] and there's an examples directory? I assume "examples" would be run before "python", which is probably wrong.

I feel like we need to either document that people using sconscriptOrder have to specify all directories in their product, or explicitly manage all the ones they don't specify